vminfo_parser/vminfo_parser.py

```diff
@@ -315,30 +334,31 @@ def print_site_usage(self: t.Self, resource_list: list, dataFrame: pd.DataFrame)
 
 
 class Visualizer:
-    def __init__(self: t.Self, analyzer: A) -> None:
-        self.analyzer = analyzer
+    def __init__(
+        self: t.Self,
+        dataFrame: pd.DataFrame,
+        column_headers=None,
+    ) -> None:
         self.cli_output = CLIOutput()
+        self.dataFrame = dataFrame
+        self.column_headers = column_headers
```

Comment:
it appears neither the dataFrame or the column_headers from the constructor are ever used.  Where those objects get used, they are passed though as arguments, which is the better behavior.  I think we can drop them from the contractor.